lib/integrations/message_formatter.rb

```diff
@@ -31,6 +31,10 @@ def run_test_failure_message
       "has a failed a test!"
     end
 
+    def fallback_text
+      "retreat!"
```

Comment:
The fallback text should be the same message but with a text-only options (no markup/hyperlinks of any sort). This was not stated in the issue and I apologize :)